comments.php

```diff
@@ -17,59 +17,61 @@
 <div id="comments" class="comments-area">
 
 	<?php
+	$front_page_id = get_option( 'page_on_front' );
+
 	comment_form( array(
 		'fields' => array(
 			'author' => '<p class="comment-form-author"><label for="author">Name <span class="optional">(Optional)</span></label> <input id="author" name="author" value="" maxlength="245" aria-required="true" required="required" type="text"></p>',
 			'email' => '<p class="comment-form-email"><label for="email">Email Address <span class="optional">(Optional)</span></label> <input id="email" name="email" value="" maxlength="100" aria-describedby="email-notes" aria-required="true" required="required" type="email"></p>',
+			),
```

Comment:
Ideally, the spacing change isn't introduced here so that the diff is cleaner. It doesn't seem like much is functionally changing, just an extra tab. `comment_field`, etc... should probably line up with `fields`.